WISE-Community/WISE-Client
# Change the add simulation view to display cards

Make sure the new card view for add simulation works properly.

You can test it locally using the instructions below, or if it's too much work to set up locally, you should be able to test it on the private client instance since the simulationProjectId should already be set up with the simulations and thumbnails on production.

Here is my simulation project. Make sure to change the simulationProjectId in your database portal structures value if you need to.
[84.zip](https://github.com/WISE-Community/WISE-Client/files/6995700/84.zip)

Here is where you can download the files for the first simulation for the sake of testing
https://wise.berkeley.edu/curriculum/shared/solar-radiation-sunrays/v1/solar-radiation-sunrays.html
https://wise.berkeley.edu/curriculum/shared/solar-radiation-sunrays/v1/thumbnail.jpg

Here is where you should add a shared simulation folder/files on your local instance for the first simulation
```/WISE-API/src/main/webapp/curriculum/shared/solar-radiation-sunrays/v1/solar-radiation-sunrays.html```
```/WISE-API/src/main/webapp/curriculum/shared/solar-radiation-sunrays/v1/thumbnail.jpg```

If there are problems accessing any of the simulation files through your local instance, you may need to change the permissions on ```solar-radiation-sunrays.html``` or ```thumbnail.jpg.```

Here are the steps to test
1. Open a unit in the Authoring Tool
2. Add a step
3. Choose to add an "Interactive Simulation"
4. Make sure the thumbnail works
5. Make sure filtering works
6. Make sure previewing a simulation works
7. Select a simulation
8. Add it to the unit and make sure the simulation works

Closes #289
